Chainlit/literalai-python
# Add "add generation" in the SDK

## Changes

- Update the `Dataset` class definition
- Add the the new mutation `addGenerationToDataset`
- Update the existing queries to fetch a "Dataset" type
- Add test

## How to test?

- Make sure to use the branch `clement/eng-957-add-dataset-item-from-generation` on your local server
- Notice that the tests are passing ✅
